### DISCUSSION TITLE
Add Export to Shapefile from Observations

### DISCUSSION THREAD

**Discussion by aliya-ryan** [digidem/mapeo-desktop#594] [Ideas]
**Describe the idea**
It would be good for Mapeo Desktop to export to shapefile format from the Observations View.

**What's your role? Where and how do you currently use the application?**
Request came from Luke Weiss using Mapeo to do territory mapping work in the Amazon with communities.

**What is your ideal step by step workflow?**
ArcMap does not automatically accept geojsons, you have to pay for another plugin to take geojson and then be able to convert to shapefile. Therefore to have a direct export to shapefile function would be good.

**Additional context**
To get around this problem last week Luke did a geojson export, and then sent this to Aliya, I converted it to shapefile in QGIS (which Luke didn't have) and sent it back. There are probably other workarounds and free converters, but a direct export would be best (of either whole data or filtered data).

**Comment by rudokemper** [comment]
Christopher Bartlett from the Santo Sunset Environmental Network also mentioned that KMZ / KML would be good for their workflow collating data from Mapeo with other sources in Google Earth Pro. Right now they have been downloading GeoJSON and using an online converter, which is not ideal since they are frequently out of connectivity.
